src/Connection.js

```diff
@@ -0,0 +1,52 @@
+
+define(["ObjectsConfig"], function (ObjectsConfig) {
+    "use strict";
+
+    var Factory = function () {
```

Comment:
just add a factory method createConnection
Easier to use in this case than Factory.createConnection